src/index.html

```diff
@@ -1,13 +1,35 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img src="/src/images/imac.jpeg" alt="image" class="card__img">
+      <h1 class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+      <p class="card__code">Код товара: 195434</p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__rating-count">Отзывов: 5</div>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">Цена:</p>
+        <p class="card__price-amount">69 999 грн</p>
+      </div>
+      <button href="#" class="card__button" data-qa="hover">КУПИТЬ</button>
```

Comment:
href attribute for button? 🤔

```suggestion
      <a
        href="#"
        class="card__button"
        data-qa="hover"
      >
        КУПИТЬ
      </a>
```

Reply:
Dear Alina=)

Could you please tell me, why I should change <button> to <a>?

In the task, it should be a button, or I am wrong?

Reply:
🙃
<img width="472" alt="Screenshot 2022-07-19 at 15 38 03" src="https://user-images.githubusercontent.com/47606379/179752034-cdffe28b-e3a9-41f1-af02-69683df78dba.png">